### ISSUE TITLE
JS injection can't access global variables / objects

### ISSUE THREAD

**Issue by llabusch93** [nativefier/nativefier#1269]
Hello folks,

I am using the most recent version of nativefier: 
```
> nativefier --version 45.0.0
```
When I inject a JS file like this one:

```
(function () {

    function disableTracking() {
        odoo.session_info.user_context.tracking_disable = true;
    };

   disableTracking();

})();
```

and build the app with the following command:

```
nativefier --name "<APP_NAME>" --conceal --single-instance --inject test.js --lang en <APP_URL> .local/share/nativefier-apps
```

The console of the app tells me, that the global variable _odoo_ is not defined:

```
Error encoutered injecting JS files ReferenceError: odoo is not defined
    at disableTracking (VM128 inject-c5c0b1.js:4)
    at VM128 inject-c5c0b1.js:7
    at Object.<anonymous> (VM128 inject-c5c0b1.js:9)
    at Object.<anonymous> (VM128 inject-c5c0b1.js:10)
    at Module._compile (VM43 loader.js:1078)
    at Object.Module._extensions..js (VM43 loader.js:1108)
    at Module.load (VM43 loader.js:935)
    at Module._load (VM43 loader.js:776)
    at Function.f._load (VM70 asar_bundle.js:5)
    at Function.o._load (VM75 renderer_init.js:33)
```

From the documentation I expect that this variable should be available:

> Note about JS injection: injected JS is loaded after DOMContentLoaded, so you can assume the DOM is complete & available.

I also tried various manual checkings in the JS file for a loaded DOM, all with the same result. The variable is available in the applications console tough:

```
> odoo
{csrf_token: "f630ede6b4ccdd28d4b51879292f2e06be7bdb71o", session_info: {…}, testing: false, debug: true, remaining_jobs: Array(0), …}
csrf_token: "f630ede6b4ccdd28d4b51879292f2e06be7bdb71o"
debug: true
define: ƒ ()
log: ƒ ()
process_jobs: ƒ (jobs,services)
remaining_jobs: []
session_info: {session_id: "145d5d94b26f8426bc57829a4ac44bed5e0f31f9", uid: 44, is_system: true, is_superuser: false, user_context: {…}, …}
testing: false
__DEBUG__: {didLogInfo: {…}, get_dependencies: ƒ, get_dependents: ƒ, get_waited_jobs: ƒ, get_missing_jobs: ƒ, …}
_modules: (126) ["base", "web", "web_editor", "web_planner", "bus", "web_tour", "mail", "portal", "website", "product", "account", "payment", "website_payment", "sales_team", "sale", "account_invoicing", "sale_management", "wk_appointment", "website_appointment", "website_seo_pro", "calendar", "contacts", "crm", "project", "barcodes", "stock", "hr", "hr_timesheet", "hr_holidays", "website_partner", "website_form", "website_crm", "hr_attendance", "document", "hr_recruitment", "hr_expense", "account_bank_statement_import", "account_bank_statement_import_ofx", "account_cutoff_base", "date_range", "report_xlsx", "account_financial_report", "account_lock_date_update", "account_reversal", "archeti_external_layout_clean", "archeti_website_appointment", "attachment_large_object", "attachment_preview", "auth_jwt", "auth_signup", "base_import", "board", "contract", "crm_timesheet", "currency_rate_update", "customer_outstanding_statement", "knowledge", "document_page", "document_page_approval", "document_page_reference", "document_url", "email_template_qweb", "git_webhooks", "iap", "knowsystem", "knowsystem_multilang", "l10n_ca", "mail_activity_creator", "mail_inline_css", "mail_outbound_static", "mail_tracking", "web_kanban_gauge", "mass_mailing", "mail_tracking_mass_mailing", "mass_editing", "meeting_ida", "web_widget_color", "mis_builder", "survey", "oh_appraisal", "partner_sector", "payment_paypal", "payment_transfer", "portal_task_kanban", "sale_timesheet", "project_report_task_overview", "project_stage_state", "project_task_add_very_high", "project_task_default_stage", "project_task_dependency", "web_timeline", "project_timeline", "project_timeline_hr_timesheet", "project_timeline_task_dependency", "rating", "sale_stock", "web_ir_actions_act_multi", "web_ir_actions_act_view_reload", "timesheet_force_on_change_state", "web_diagram", …]
__proto__: Object
```

So what am I doing wrong? Is something special needed to access global variables?

Any help would be very much appreciated.

Thanks in advance!

**Comment by insolite** [comment]
It seems to be fixed with https://github.com/nativefier/nativefier/commit/ca7d25f4327d353d940cb51b0e419d15df8b655b#diff-60366959d6497ad3a50fdc7b32dd59c76eb35ab6c0ea18f0e700b15c03b67b12R151 in v45.0.6.

**Comment by ronjouch** [comment]
> It seems to be fixed with [ca7d25f#diff-60366959d6497ad3a50fdc7b32dd59c76eb35ab6c0ea18f0e700b15c03b67b12R151](https://github.com/nativefier/nativefier/commit/ca7d25f4327d353d940cb51b0e419d15df8b655b#diff-60366959d6497ad3a50fdc7b32dd59c76eb35ab6c0ea18f0e700b15c03b67b12R151) in v45.0.6.

@insolite wheee! Thanks for the precise report. With that, closing.
